foreman_fog_proxmox.gemspec

```diff
@@ -36,6 +36,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'deface'
   s.add_dependency 'fog-proxmox', '~> 0.13'
   s.add_development_dependency 'rdoc'
-  s.add_development_dependency 'rubocop'
+  s.add_development_dependency 'rubocop', '~> 0.80.0'
```

Comment:
rubocop min version was not specified then it took last version suitable

Reply:
Yes, I adapted it form https://github.com/theforeman/foreman_remote_execution/blob/03eb17de3a88bfc1d27a3bbf36fb3c0b05bbe4c6/foreman_remote_execution.gemspec#L32 so that we know for sure, which version works and it doesn't break if a new rubocop version is released.